junit-jupiter-engine/src/test/java/org/junit/jupiter/api/condition/DisabledIfSystemPropertyIntegrationTests.java

```diff
@@ -26,53 +31,66 @@
  */
 class DisabledIfSystemPropertyIntegrationTests {
 
-	private static final String KEY = "DisabledIfSystemPropertyTests.key";
-	private static final String ENIGMA = "DisabledIfSystemPropertyTests.enigma";
-	private static final String BOGUS = "DisabledIfSystemPropertyTests.bogus";
+	private static final String KEY1 = "DisabledIfSystemPropertyTests.key1";
+	private static final String KEY2 = "DisabledIfSystemPropertyTests.key2";
+	private static final String ENIGMA = "enigma";
+	private static final String BOGUS = "bogus";
 
 	@BeforeAll
-	static void setSystemProperty() {
-		System.setProperty(KEY, ENIGMA);
+	static void setSystemProperties() {
+		System.setProperty(KEY1, ENIGMA);
+		System.setProperty(KEY2, ENIGMA);
 	}
 
 	@AfterAll
-	static void clearSystemProperty() {
-		System.clearProperty(KEY);
+	static void clearSystemProperties() {
+		System.clearProperty(KEY1);
+		System.clearProperty(KEY2);
 	}
 
 	@Test
 	@Disabled("Only used in a unit test via reflection")
 	void enabledBecauseAnnotationIsNotPresent() {
```

Comment:
Should be `disabledBecauseAnnotationIsNotPresent` right?

Reply:
The name is actually correct, but... I fully understand how that's confusing. 😉 

It should be "enabled" because `@DisabledIfSystemProperty` is not present on the test method. The corresponding unit test `org.junit.jupiter.api.condition.DisabledIfSystemPropertyConditionTests.enabledBecauseAnnotationIsNotPresent()` verifies that. Note that the unit test ignores the presence of `@Disabled`.

Does that make sense now?

Reply:
Having said that, the `@Disabled` declaration and use of `fail()` are unnecessary for that test method, so I'll remove them. 👍

Reply:
I think my comment got lost, but... I removed the `@Disabled` annotation and use of `fail()` in that method since they didn't make sense.

Reply:
Ah, I see.
This whole enabled/disabled thing everywhere is pretty disorienting ahah 😅